spec/lib/dor/assembly/stub_content_metadata_parser_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe Dor::Assembly::StubContentMetadataParser do
+  describe '#stub_content_metadata_parser' do
+    let(:druid) { 'druid:aa111bb3333' }
+    let(:item) { Dor::Assembly::Item.new(druid: druid) }
+
+    it 'maps content metadata types to the gem correctly' do
+      ['flipbook (r-l)', 'book', 'a book (l-r)'].each do |content_type|
```

Comment:
I think these should be broken out into separate context blocks.  "When stub_object_type is book", etc.

Reply:
split them all out